db/migrate/20210910121223_create_uservotes.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUservotes < ActiveRecord::Migration[6.1]
+  def change
+    create_table :uservotes do |t|
```

Comment:
using snake_case for model naming conventions 
reference link for naming conventions https://guides.rubyonrails.org/active_record_basics.html

Reply:
user_votes